server_2.R

```diff
@@ -0,0 +1,736 @@
+# Required Packages
+{
+  if (!require("shinydashboard")) install.packages("shinydashboard")
+  if (!require("jpeg")) install.packages("jpeg")
+  if (!require("RPostgres")) install.packages("shinydashboard")
+  if (!require("formattable")) install.packages("formattable")
+  if (!require("plotly")) install.packages("plotly")
+  if (!require("purrr")) install.packages("purrr")
+  if (!require('ggcharts')) install.packages("ggcharts") 
+  
+  # data prep
+  library(tidyr)
+  library(dplyr)
+  library(reshape2)
+  library(rlang)
+  library(purrr)
+  library(lubridate)
+  library(glue)
+  library(splitstackshape)
+  library(devtools)
+  library(patchwork)
+  # visualizations
+  library(ggplot2)
+  library(ggcharts)
+  library(plotly)
+  library(formattable)
+  # db connections
+  library(RPostgreSQL)
+  library(RPostgres)
+  # shiny
+  library(shiny)
+  library(shinyWidgets)
+  library(shinythemes)
+  library(shinydashboard)
+}
+
+# UI elements config
+{
+  cols = list(yellow='#ffeaa7',blue='#74b9ff',dark_yellow='#fdcb6e',dark_blue='#0984e3')  
+  params <- list(font = 'sans',fontface = 'plain')
+}
+
+# DB connection
+mta_con <- dbConnect(Postgres(),
+                     user     = "fzgxltqkgmaklf",
+                     password = "6ad610f8f95f1f570ad6c846b68e74f0d692386a8e43d2fce5976f1718e2b779",
+                     dbname   = "d5m2p6kka0vf8d",
+                     port     = "5432",
+                     host     = "ec2-184-73-232-93.compute-1.amazonaws.com",
+                     sslmode = 'require')
+
+# DB calls - fetch data
+{
```

Comment:
regarding the sqls - I would create a script something like sqls_object.r which contains the following -
```
sqls_per_segmemt <- list(games = "select..",
events = "select..",
etc)
```

Then - 

```
data_per_segment <- purrr::map(sqls_per_segmemt, ~ dbGetQuery(.x, conn = mta_con)
```